packages/trader/src/App/Components/Elements/PositionsDrawer/positions-drawer.jsx

```diff
@@ -65,7 +71,7 @@ class PositionsDrawer extends React.Component {
                                 openContract={openContract}
                                 key={portfolio_position.id}
                                 currency={currency}
-                                toggleUnsupportedContractModal={toggleUnsupportedContractModal}
+                                toggleUnsupportedContractModal={this.setUnSupportedContractModal}
```

Comment:
nit: Did you mean `this.setUnsupportedContractModal`?

Reply:
On it.